samples/Hardware.GiantGecko/README.md

```diff
@@ -0,0 +1,37 @@
+# 🌶️ to 🌶️🌶️ - Hardware Giant Gecko sample pack
```

Comment:
I guess here, only 1 spice :-)

Reply:
You're correct. I was lazy and left it there for the other ones that are inbound... 😅